core/domain/exp_jobs_one_off.py

```diff
@@ -488,6 +488,9 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(exp_summary_model):
+        # This is an only way to remove the field from the model,
```

Comment:
Sorry -- could you change "an" to "the" here and below?

Reply:
Done.